Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -14,6 +14,7 @@ public protocol SettingsAccountViewModelOutputs {
 
   var reloadData: Signal<Void, NoError> { get }
   var transitionToViewController: Signal<UIViewController, NoError> { get }
+  var goToAddCard: Signal<Void, NoError> { get }
```

Comment:
Did we want to add a test for this output?

Reply:
Not yet, this output is actually going to be removed  and go in the payments method screen when we get to that. It's only here to get to the `Add New Card` screen. We can actually remove this?